www/js/app.js

```diff
@@ -819,47 +821,45 @@ define(['jquery', 'zimArchiveLoader', 'util', 'uiUtil', 'cookies','abstractFiles
             // Create (or replace) the "base" tag with our base URL
             $('#articleContent').contents().find('head').find("base").detach();
             $('#articleContent').contents().find('head').append("<base href='" + baseUrl + "'>");
+            
+            var currentProtocol = location.protocol;
+            var currentHost = location.host;
 
             // Convert links into javascript calls
             $('#articleContent').contents().find('body').find('a').each(function() {
+                var href = $(this).attr("href");
                 // Compute current link's url (with its namespace), if applicable
-                var url = regexpZIMUrlWithNamespace.test(this.href) ? this.href.match(regexpZIMUrlWithNamespace)[1] : $(this).attr("href");
-                if (url === null || url === undefined) {
-                    return;
+                var zimUrl = regexpZIMUrlWithNamespace.test(this.href) ? this.href.match(regexpZIMUrlWithNamespace)[1] : "";
```

Comment:
Good that you renamed this to `zimUrl`, it makes the subsequent code much clearer.